doc/python/lines-on-maps.md

```diff
@@ -111,6 +111,74 @@ fig.update_layout(
 
 fig.show()
 ```
+### Performance improvement: put many lines in the same trace
+For very large amounts (>1000) of lines, performance may become critcal. If you can relinquish setting individual line styles (e.g. opacity), you can put multiple paths into one trace. This makes the map render faster and reduces the script execution time and memory consumption. 
+
+Use ```None``` between path coordinates to create a break in the otherwise connected paths. 
+
+```python 
+import plotly.graph_objects as go
+import pandas as pd
+
+df_airports = pd.read_csv('https://raw.githubusercontent.com/plotly/datasets/master/2011_february_us_airport_traffic.csv')
+df_airports.head()
+
+df_flight_paths = pd.read_csv('https://raw.githubusercontent.com/plotly/datasets/master/2011_february_aa_flight_paths.csv')
+df_flight_paths.head()
+
+fig = go.Figure()
+
+fig.add_trace(go.Scattergeo(
+    locationmode = 'USA-states',
+    lon = df_airports['long'],
+    lat = df_airports['lat'],
+    hoverinfo = 'text',
+    text = df_airports['airport'],
+    mode = 'markers',
+    marker = dict(
+        size = 2,
+        color = 'rgb(255, 0, 0)',
+        line = dict(
+            width = 3,
+            color = 'rgba(68, 68, 68, 0)'
+        )
+    )))
+
+flight_paths = []
+lons = []
+lats = []
+for i in range(len(df_flight_paths)):
+    lons += [df_flight_paths['start_lon'][i], df_flight_paths['end_lon'][i], None] 
+    lats += [df_flight_paths['start_lat'][i], df_flight_paths['end_lat'][i], None]
```

Comment:
```suggestion
import numpy as np
lons = np.empty(3 * len(df_flight_paths))
lons[::3] = df_flight_paths['start_lon']
lons[1::3] = df_flight_paths['end_lon']
lons[::3] = None
lats = np.empty(3 * len(df_flight_paths))
lats[::3] = df_flight_paths['start_lat']
lats[1::3] = df_flight_paths['end_lat']
lats[::3] = None
```

Reply:
Since this example is about performance, if you have a large number of lines using arrays will be much faster than the for loop (for 10000 lines the for loop takes almost one second on my machine).